dasharo-compatibility/coreboot-fan-control.robot

```diff
@@ -32,7 +32,9 @@ CFN001.001 CPU temperature and fan speed can be read (Debian 11.02)
     ...    is available and can be read.
     Skip If    not ${TESTS_IN_DEBIAN_SUPPORT}    CFN001.001 not supported
     Power On
-    Boot From USB
+    ${usb}=    Get USB Boot Option
+    Power On
```

Comment:
Why do we need two power ons?

Reply:
` Boot System Or From Connected Disk ` assumes that the platform is rebooted (it launches the boot menu again). I could add a new keyword that boots the specified option from the boot menu without the need to restart. It would then make sense to wrap all of this in a single keyword, which would pretty much reintroduce the `Boot From USB` keyword. Is that okay?

Reply:
It seems the idea in the original MR (https://github.com/Dasharo/open-source-firmware-validation/pull/92/files) was to use existing kwd for booting a menu entry, by simply using another argument. But it has not been completed as we can see. The assumption there was that the USB has static name defined in config IIUC, which should be fine in the most cases. We can also try to dynamically find this entry as you suggest. We should consider also what happens if we have multiple USB drives in this case.

Reply:
This is exactly what I'm doing here, but as of now it requires a restart, since the `Boot From ...` keyword launches the boot menu. I don't see how we can boot from USB without restarting if we don't want to create a new keyword.